Paymetheus/ViewModels/PurchaseTicketsViewModel.cs

```diff
@@ -0,0 +1,404 @@
+                {
+                    _number = 0;
+                    string errorString = "Not enough funds; have " +
+                        _selectedAccount.Balances.SpendableBalance.ToString() + " want " +
+                        ((Amount)(_stakeDifficultyProperties.Price * value)).ToString();
+                    throw new ArgumentException(errorString);
+                }
+
+                _number = value;
+                _settingsAreValid();
+            }
+        }
+
+        // The default expiry is 16.
+        private int _expiry = 16;
+        private int _minExpiry = 2;
+        public int Expiry
+        {
+            get { return _expiry; }
+            set
+            {
+                if (value <= _minExpiry)
+                {
+                    _expiry = 0;
+                    throw new ArgumentException("Expiry must be a minimum of 2 blocks");
+                }
+
+                _expiry = value;
+                _settingsAreValid();
+            }
+        }
+
+        // TODO Declare this as a global somewhere?
+        private Amount _minFeeAmount = Denomination.Decred.AmountFromDouble(0.01000000);
+        private Amount _maxFeeAmount = Denomination.Decred.AmountFromDouble(0.99999999);
+
+        private Amount _splitFee = 0;
+        public double SplitFee
+        {
+            get { return Denomination.Decred.DoubleFromAmount(_splitFee); }
+            set
+            {
+                var _testAmount = Denomination.Decred.AmountFromDouble(value);
+
+                if (_testAmount < _minFeeAmount)
+                {
+                    _splitFee = 0;
+                    throw new ArgumentException("Too small fee passed (must be >= 0.01000000 DCR/KB)");
+                }
+
+                if (_testAmount > _maxFeeAmount)
+                {
+                    _splitFee = 0;
+                    throw new ArgumentException("Too big fee passed (must be <= 0.99999999 DCR/KB)");
+                }
+
+                _splitFee = _testAmount;
+                _settingsAreValid();
+            }
+        }
+
+        private Amount _ticketFee = 0;
+        public double TicketFee
+        {
+            get { return Denomination.Decred.DoubleFromAmount(_ticketFee); ; }
+            set
+            {
+                var _testAmount = Denomination.Decred.AmountFromDouble(value);
+
+                if (_testAmount < _minFeeAmount)
+                {
+                    _ticketFee = 0;
+                    throw new ArgumentException("Too small fee passed (must be >= 0.01000000 DCR/KB)");
+                }
+
+                if (_testAmount > _maxFeeAmount)
+                {
+                    _ticketFee = 0;
+                    throw new ArgumentException("Too big fee passed (must be <= 0.99999999 DCR/KB)");
+                }
+
+                _ticketFee = _testAmount;
+                _settingsAreValid();
+            }
+        }
+
+        private void _settingsAreValid()
```

Comment:
The _ prefix should only be used for private fields, not functions.  And to make this clearer (the current name makes it sound like it returns a boolean expression whether they are valid or not) i would name this EnableOrDisableSendCommand.